### PR TITLE
Remove --commit-from and --commit-to from docs

### PR DESCRIPTION
**Pull request by emmahsax**
### Description:
Removing outdated flags `--commit-from` and `--commit-to` from the `README.md` documentation.

And my apologies, my text editor took off all of the trailing whitespace in the `README.md` file.

### Checklist:

* [ ] ~~Does your PR pass tests?~~
* [ ] ~~Have you written new tests for your changes?~~
* [ ] ~~Have you lint your code locally prior to submission?~~

### CODE REVIEW

**Review by w0rmr1d3r — APPROVED** [review]

**Comment by w0rmr1d3r** [comment]
Looks good to me, let's wait for @zricethezav approval

**Comment by zricethezav** [comment]
@emmahsax thank you 🙇🏻

**Pull request merged**
